enterprise/api/v1/views/enterprise_group.py

```diff
@@ -117,11 +124,29 @@ def get_learners(self, *args, **kwargs):
             }
 
         """
+        query_params = self.request.query_params.copy()
+        is_reversed = False
+        if sort_by := query_params.get('sort_by'):
+            is_reversed = sort_by[0] == '-'
+            sort_by = utils.convert_to_snake(sort_by.lstrip('-'))
+            query_params['sort_by'] = sort_by
```

Comment:
wouldn't things be much simpler if you accepted an optional, boolean `reversed` query parameter?